### ISSUE TITLE
Package binding hides a useful evil binding

### ISSUE THREAD

**Issue by ambihelical** [ninrod/evil-string-inflection#3]
The binding for g~ defined by evil (evil-invert-case) is actually useful.  I bind evil-operator-string-inflection to g- which is a great mnemonic, easier to type and currently unused in evil.  However g~ is still lost unless I rebind it.  Consider a different binding or making the default binding optional.

**Comment by ninrod** [comment]
interesting. `g~` should not be lost if you rebind it. let me try to reproduce it.

**Comment by ambihelical** [comment]
Yes, absolutely.  I *can* rebind it, but I think the package should make it easier to avoid having to do this.

**Comment by ninrod** [comment]
Well, this should be easy, but apparently I cannot make it work, because I'm horrible at elisp.

So I've posted a [question](https://stackoverflow.com/q/51901729/4921402) in SO. If this questions gets answered, or if somehow I find the solution, I'll quickly fix it.

**Comment by ambihelical** [comment]
I am for sure no better an elisp programmer. But I think you should be able to make a defcustom for the key, with default value of "g~", or (kbd "g~") if you want and then pass that variable to define-key.   I don't think kbd is really needed except for some kinds of key definitions, but I am not sure.

**Comment by VanLaser** [comment]
I tested with this code, which works. Emacs 26.1:

```elisp
(defcustom evil-test-key  "g~"
  "Key for a test text object."
  :type 'string)

(setq evil-test-key "g~")
(define-key evil-normal-state-map (kbd evil-test-key) 'evil-operator-eval)
```

**Comment by abeluck** [comment]
I ran into this today as well. I also use the defailt `g~` for  invert case.

**Comment by ninrod** [comment]
@VanLaser, you are doing a rebind on the client config, yes?
